content/ja/docs/concepts/cluster-administration/networking.md

```diff
@@ -0,0 +1,293 @@
+
+The Nuage platform uses overlays to provide seamless policy-based networking between Kubernetes Pods and non-Kubernetes environments (VMs and bare metal servers). Nuage's policy abstraction model is designed with applications in mind and makes it easy to declare fine-grained policies for applications.The platform's real-time analytics engine enables visibility and security monitoring for Kubernetes applications.
+
+### OpenVSwitch
+
+[OpenVSwitch](https://www.openvswitch.org/) is a somewhat more mature but also
+complicated way to build an overlay network.  This is endorsed by several of the
+"Big Shops" for networking.
+
+### OVN (Open Virtual Networking)
+
+OVN is an opensource network virtualization solution developed by the
+Open vSwitch community.  It lets one create logical switches, logical routers,
+stateful ACLs, load-balancers etc to build different virtual networking
+topologies.  The project has a specific Kubernetes plugin and documentation
+at [ovn-kubernetes](https://github.com/openvswitch/ovn-kubernetes).
+
+### Project Calico
+
+[Project Calico](http://docs.projectcalico.org/) is an open source container networking provider and network policy engine.
+
+Calico provides a highly scalable networking and network policy solution for connecting Kubernetes pods based on the same IP networking principles as the internet, for both Linux (open source) and Windows (proprietary - available from [Tigera](https://www.tigera.io/essentials/)).  Calico can be deployed without encapsulation or overlays to provide high-performance, high-scale data center networking.  Calico also provides fine-grained, intent based network security policy for Kubernetes pods via its distributed firewall.
+
+Calico can also be run in policy enforcement mode in conjunction with other networking solutions such as Flannel, aka [canal](https://github.com/tigera/canal), or native GCE, AWS or Azure networking.
+
+### Romana
+
+[Romana](http://romana.io) is an open source network and security automation solution that lets you deploy Kubernetes without an overlay network. Romana supports Kubernetes [Network Policy](/docs/concepts/services-networking/network-policies/) to provide isolation across network namespaces.
+
+### Weave Net from Weaveworks
+
+[Weave Net](https://www.weave.works/products/weave-net/) is a
+resilient and simple to use network for Kubernetes and its hosted applications.
+Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-plugin/)
+or stand-alone.  In either version, it doesn't require any configuration or extra code
+to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+The early design of the networking model and its rationale, and some future
+plans are described in more detail in the [networking design
+document](https://git.k8s.io/community/contributors/design-proposals/network/networking.md).
```

Comment:
This is not yet translated.

Reply:
@sftim

This is on purpose. The PR message roughly translated as "Just where I wanted" :)

For the Japanese translation, we allow partial translation.

Reply:
Thanks, applied the translation.

Reply:
Ah, OK.